.claude/commands/ticket.md

```
@@ -53,6 +53,20 @@ Update an existing ticket.
 /ticket update PROJ-123 --label "High Risk"
 ```
 
+### cycle
+Manage cycles (time-boxed sprints/iterations of issues for the team).
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)